Content.Server/Medical/Components/HealthAnalyzerComponent.cs

```diff
@@ -17,6 +17,17 @@ public sealed partial class HealthAnalyzerComponent : Component
         [DataField("scanDelay")]
         public float ScanDelay = 0.8f;
 
+        /// <summary>
+        /// Which entity has been scanned, for continuous updates
+        /// </summary>
+        public EntityUid? ScannedEntity;
+
+        /// <summary>
+        /// The maximum range at which the analyser can read an entities vitals
+        /// </summary>
+        [DataField("maxScanRange")]
+        public float MaxScanRange = 5f;
```

Comment:
I wanted to make allowances for having to move around medbay (for supplies or chems)
5 tiles was enough to walk back and forth between most beds and where the chems are placed (curse you origin)
I'll test 1.5

Reply:
allowing such a long range has a lot of issues that come with it since scanners are not used only by doctors to scan health, now, i dont think the window closes when you get out of range so if you want a recordatory of the list of damage when looking for meds you can just keep it open

Reply:
The minimum range i can reasonably choose is ~2.5.
Any less than that and its possible to scan a patient from beyond the range limit - and it instantly getting disabled on the next Update()

Reply:
On review i think the comment is misleading.
The range to scan a patient is handeled elsewhere - interaction range.
This field controls the range at which continuous updates are received - ie after scanning you could move up to 5 tiles away before the updates stop.

I have updated the comment to be more accurate